packages/react/src/drafts/ActionBar/ActionBar.tsx

```diff
@@ -254,7 +254,7 @@ export const ActionBar: React.FC<React.PropsWithChildren<ActionBarProps>> = prop
   return (
     <ActionBarContext.Provider value={{size, setChildrenWidth}}>
       <Box ref={navRef} sx={getNavStyles()}>
-        <NavigationList sx={ulStyles} ref={listRef} role="list">
+        <NavigationList sx={listStyles} ref={listRef} role="toolbar">
```

Comment:
Nice, toolbar makes total sense here. I tested the arrow focusing and it looks fine to me 👍